File: Modul 6/PRAK603/PRAK603-2310817310011-MuhammadRizky.py
a, b = map(int, input().split())
matrixAtas = []; matrixBawah = []

if a == b:
    x = list(map(int, input().split()))
    y = list(map(int, input().split()))

    for i in range(a):
        matrixAtas.append(x[i])

    for i in range(b):
        matrixBawah.append(y[i])

    for i in range(b):
        hasil = matrixAtas[i] * matrixBawah[i]
        print(hasil, end = " ")

else:
    print("Jumlah tidak sama")

File: Modul 6/PRAK603/PRAK603-2310817310011-MuhammadRizky.c
#include <stdio.h>

int main(){
    int a, b, matrix;
    scanf("%d %d", &a, &b);
    int x[a], y[b];

    if (a == b){
        for (int i = 0; i < a; i++){
            scanf("%d", &x[i]);
        }

        for (int i = 0; i < b; i++){
            scanf("%d", &y[i]);
        }

        for (int i = 0; i < a; i++){
            int hasil = x[i] * y[i];
            printf(" %d ", hasil);
        }

    } else {
        printf("Jumlah tidak sama");
    }
    return 0;
}
